keepalived/vrrp/vrrp.c

```diff
@@ -1544,6 +1544,7 @@ vrrp_state_master_rx(vrrp_t * vrrp, char *buf, int buflen)
 		   (hd->priority == vrrp->effective_priority &&
 		    addr_cmp < 0)) {
 		/* We receive a lower prio adv we just refresh remote ARP cache */
+		vrrp_log_effective_priority (vrrp);
 		log_message(LOG_INFO, "VRRP_Instance(%s) Received lower prio advert %d"
 				      ", forcing new election", vrrp->iname, hd->priority);
```

Comment:
Could the `vrrp_log_effective_priority()` and the `log_message()` be merged into a single log entry by not calling
`vrrp_log_effective_priority()` and changing the log_message to be:
```
log_message(LOG_INFO, "VRRP_Instance(%s) Received lower prio advert %d, our priority %d",
                       vrrp->iname, hd->priority, vrrp->effective_priority);
```